pkgs/development/python-modules/pygerber/default.nix

```
@@ -0,0 +1,105 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pythonOlder,
+  # build-system
+  poetry-core,
+
+  # dependencies
+  numpy,
+  click,
+  pillow,
+  pydantic,
+  pyparsing,
+  typing-extensions,
+
+  # optional dependencies
+  pygls,
+  lsprotocol,
+  drawsvg,
+  pygments,
+  shapely,
+
+  # test
+  filelock,
+  dulwich,
+  tzlocal,
+  pytest-xdist,
+  pytest-cov,
+  pytest-lsp,
+  pytest-asyncio,
+  pytest-mock,
+  pytestCheckHook,
+
+}:
+
+buildPythonPackage rec {
+  pname = "pygerber";
+  version = "2.4.2";
+  pyproject = true;
+
+  disabled = pythonOlder "3.8";
+
+  src = fetchFromGitHub {
+    owner = "Argmaster";
+    repo = pname;
+    rev = "refs/tags/v${version}";
```

```suggestion
    repo = "pygerber;
    tag = "v${version}";
```
Avoid using pname #277994.
The tag parameter saves some boilerplate. (you may need to rebase if your branch does not support it)